site/content/blog/2019-12-20-onboarding-with-mattermost.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Onboarding with Mattermost"
+slug: onboarding-with-mattermost
+date: 2019-12-20T00:00:00-04:00
+categories:
+    - "announcements"
+author: CJ Wild
+github: imisshtml
+community: claude.wild
+---
+
+I recently joined Mattermost as a Remote Software Engineer and while I’ve worked in remote environments before, I was really surprised at how amazing this company is to work with. During my onboarding I discovered they have really developed a culture built around being Remote; one that allowed for me to feel as part of a larger team and always engaged, rather than separated from my peers and working alone. 
+
+### Sponsored Simulation
```

Comment:
For each section, would there be a public reference / link / page with more details? I'm thinking that if someone would like to learn more about the sponsored simulation, or mentorship, then the reader can click the link to learn more.

E.g. for sponsored simulation, we could link to https://docs.mattermost.com/process/developer.html#audition. For onboarding/checklist, we could link to https://handbook.mattermost.com/contributors/onboarding/staff-on-boarding-guide or https://developers.mattermost.com/internal/onboarding/new-staff-guide/ (not sure which is used). I can search for other resources as well, but wasn't sure which ones you had primarily used.

(If a page or link doesn't exist, no need to create one - we can just use whatever we have today).